src/Internal/PsrMessageStream.php

```diff
@@ -131,9 +131,9 @@ private function getOpenStream(): InputStream
 
     private function readFromStream(): string
     {
-        $data = coroutine(function (): ?string {
+        $data = async(function (): ?string {
             return $this->getOpenStream()->read();
-        })->await(new TimeoutCancellationToken($this->timeout));
+        })->await(new TimeoutCancellation($this->timeout));
```

Comment:
This can be changed to avoid `async` now, because read accepts an cancellation now.

Reply:
Should be all set now - sorry for the delay